sdks/java/io/google-cloud-platform/src/main/java/org/apache/beam/sdk/io/gcp/bigquery/BigQueryIO.java

```diff
@@ -687,7 +696,7 @@ public void validate(PipelineOptions options) {
                 new JobConfigurationQuery()
                     .setQuery(getQuery().get())
                     .setFlattenResults(getFlattenResults())
-                    .setUseLegacySql(getUseLegacySql()));
+                    .setUseLegacySql(getUseLegacySql()), getQueryLocation());
```

Comment:
Newline between the `JobConfigurationQuery` and the location, as it reads as though it's part of that configuration (which I don't think it is)

Reply:
Done.